admin/load.php

```diff
@@ -245,7 +253,8 @@ function perflab_get_modules( $modules_root = null ) {
 
 	$modules      = array();
 	$module_files = array();
-	$modules_dir  = @opendir( $modules_root ); // phpcs:disable WordPress.PHP.NoSilencedErrors.Discouraged
+	// PHPCS ignore reason: A modules directory is always present.
```

Comment:
Based on all of these reasons listed for silencing `opendir`, it seems like we should be able to remove the `@` silencing warnings throughout this file. Do you know if there is a reason that these were silenced initially?

Reply:
Thanks @joemcgill for asking. This change was introduced from the initial version, so may be @felixarntz will know the reason behind this?

If it is not blocker can we open a separate issue and discuss it there?

Reply:
Open at https://github.com/WordPress/performance/issues/709